core/diirt/diirt-features/pom.xml

```diff
@@ -30,7 +30,6 @@
         <!-- Plugin required to properly publish source feature -->
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-p2-plugin</artifactId>
-        <version>${tycho.version}</version>
```

Comment:
Claudio, 

Can you roll back this one change from this PR and raise a separate issue for the 'overriding managed version' warnings? This change is unrelated to the your work on the Diirt preferences.

The warnings from Eclipse may be correct and these overrides aren't required. Removing them is a change that would need to be made to all the `pom.xml` files across all the Git repos for both the `tycho.version` and `tycho-extra.version` macros and then tested in the different build systems used by the various sites.